.pre-commit-config.yaml

```diff
@@ -80,5 +80,10 @@ repos:
         files: ^cuda_pathfinder/cuda/.*\.py$  # Exclude tests directory
         args: [--config-file=cuda_pathfinder/pyproject.toml]
 
+  - repo: https://github.com/rhysd/actionlint
+    rev: "03d0035246f3e81f36aed592ffb4bebf33a03106"
```

Comment:
IIRC you need the `# frozen: x.x.x` comment otherwise pre-commit tries to update to a tag instead of a commit. The comment is apparently considered part of the format.

https://github.com/NVIDIA/cuda-python/pull/746#discussion_r2191053680